Templating/Helper/OAuthHelper.php

```diff
@@ -80,4 +81,16 @@ public function getName()
     {
         return 'hwi_oauth';
     }
+
+    /**
+     * @return Request|null
+     */
+    private function getMainRequest()
+    {
+        if (\is_callable([$this->requestStack, 'getMainRequest'])) {
```

Comment:
I think this should be just:
```suggestion
        if (\method_exists($this->requestStack, 'getMainRequest')) {
```